data/types/device/device.go

```diff
@@ -7,13 +7,23 @@ import (
 )
 
 const (
-	Type = "deviceEvent"
+	Type       = "deviceEvent"
+	StartEvent = "start"
+	StopEvent  = "stop"
 )
 
+func Events() []string {
+	return []string{
+		StartEvent,
+		StopEvent,
+	}
+}
+
 type Device struct {
 	types.Base `bson:",inline"`
 
-	SubType string `json:"subType,omitempty" bson:"subType,omitempty"`
+	SubType   string `json:"subType,omitempty" bson:"subType,omitempty"`
+	EventType string `json:"eventType,omitempty" bson:"eventType,omitempty"`
```

Comment:
Notes: 
- It should be a pointer since not all objects use it.
- This parameter is also used in deviceSensor (ok with the name "status" in the JSON ...)